### ISSUE TITLE
VS Code UI/Debugging Enchancements

### ISSUE THREAD

**Issue by starhound** [marklogic/mlxprs#244]
Good day, 

If possible for some future considerations it would be great if the ML plugin for VS Code was more developed to be in line with other modern VS Code plugins. 

Some small things that would go a long way:

* A ML Icon/Tab that provides a side bar tab: https://code.visualstudio.com/api/ux-guidelines/sidebars -- to allow people to explore the connected server, show existing modules, etc.
* A launch.json configuration for evaluation of SJS or XQY files -- without debugging them.
* Producing plugin output to a terminal window. when we evaluate queries and modules a new editor window is spawned with the result set, or its only printed out via breakpoints. 

I'm sure I'll have more recommendations as I continue to use this tool but for now those are the big 3 holding this technology back.

**Comment by mitchshepherd** [comment]
Hey @starhound, thanks for your suggestions. I'd love to discuss these further with you to better understand your use cases.  I've reached out to the email associated with your GitHub. I look forward to hearing from you.

**Comment by ghost** [comment]
bullet point 3 was my misconfiguration!

**Comment by BillFarber** [comment]
@starhound 

> A launch.json configuration for evaluation of SJS or XQY files -- without debugging them.

I've done some thinking and research about this and I'm afraid I can't find anything helpful. As far as I can tell, this simply isn't the intention of launch configurations.

Instead, I prototyped a standalone Node script that could be launched to send the request to a ML server and return the response. While this works, I've decided that it is completely impractical. As you know, this is a solved problem in the extension, but that code uses VSCode internals extensively, so I would not be able to call it from the Node script. That means I would need to duplicate a lot of code and logic. This would be particularly awkward with the code for displaying the results. However, if you would like to pursue this independently, I would be happy to share the Node script that I have created.

On the other hand, if you know of a straight-forward way to use a launch configuration to run an extension command, I would welcome a PR with an example in the docs.
